src/main/java/net/fabricmc/loom/task/RunServerTask.java

```diff
@@ -24,10 +24,17 @@
 
 package net.fabricmc.loom.task;
 
+import net.fabricmc.loom.LoomGradleExtension;
 import net.fabricmc.loom.configuration.ide.RunConfig;
 
+@Deprecated // Replaced by RunGameTask
 public class RunServerTask extends AbstractRunTask {
 	public RunServerTask() {
-		super(RunConfig::serverRunConfig);
+		super(
+				project -> {
```

Comment:
Concern two electric boogloo

Reply:
That is very concerning...

Wouldn't it be best to not use a lambada?

<:concern:707016186281590855>